src/shadowbox/server/main.ts

```diff
@@ -37,8 +37,8 @@ const MAX_STATS_FILE_AGE_MS = 5000;
 interface MetricsConfigJson {
   // Serialized ManagerStats object.
   transferStats?: ManagerMetricsJson;
-  // Serialized SharedStats object.
-  hourlyMetrics?: SharedMetricsJson;
+  // DEPRECATED: Serialized SharedStats object.
```

Comment:
Is there any reason to reference this field in code? If not, let's remove this and mention it in a comment.